pilot/pkg/bootstrap/server.go

```diff
@@ -656,29 +639,6 @@ func (s *Server) createK8sServiceControllers(serviceControllers *aggregate.Contr
 	return
 }
 
-// initMultiClusterController initializes multi cluster controller
-// currently implemented only for kubernetes registries
-func (s *Server) initMultiClusterController(args *PilotArgs) error {
-	if hasKubeRegistry(args) {
-		s.addStartFunc(func(stop <-chan struct{}) error {
```

Comment:
Should pass stop channel to `secretController` instead of define a new one.

Reply:
Will look into that approach

Reply:
Upon looking into this this should be done as a separate change because it would affect both the pilot changes, the mixer changes and the common secret controller library.   So I do not plan to make this change at this time.   Additionally there is no mechanism currently to stop or remove the secret controller once started.   It will only die when pilot also dies so the need for a stop channel at all is arguable.